documentation/modules/upgrading/proc-upgrade-cruise-control-api-version.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cruise-control-api-version-{context}']
+= Upgrading Cruise Control to support {ApiVersion}
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+
+.Procedure
+Execute the following steps for each `Kafka.spec.cruiseControl` configuration in your Kafka cluster.
+
+. Update the `Kafka` resource in an editor.
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl edit kafka _my-cluster_
+----
+
+. If `type: external` logging is configured in `.spec.cruiseControl.logging`:
++
+Replace the `name` of the ConfigMap containing the logging configuration:
++
+[source,yaml,subs="attributes+"]
+----
+logging:
+  type: external
+  name: my-config-map
+----
++
+With the `valueFrom.configMapKeyRef` field and specify both the ConfigMap `name` and the `key` under which the logging is stored:
++
+[source,yaml,subs="attributes+"]
+----
+logging:
+  type: external
+  valueFrom:
+    configMapKeyRef:
+      name: my-config-map
+      key: log4j2.properties
```

Comment:
I think this was typo in my document. In current master, this should be `log4j.properties`. Sorry.

However, it might change to `log4j2.properties` in #4487 - @kyguy @sknot-rh please advise what should we use here.

Reply:
So, apparently, we were visionary ... it will be changed to `log4j2.properties` in #4487. So we should keep this as it is.